hier.txt

```diff
@@ -0,0 +1,4 @@
+United States|Massachusetts|Suffolk County|Boston
+United States|Massachusetts|Plymouth County|Scituate
```

Comment:
Do you need the changes in this file?

Reply:
I do not think so. I'll go through the document and re create a pull request and let you know. Thanks for the help